app/car.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
```

Comment:
do you need it ?

Reply:
without classmethods probably not. thank you.